snappi_ixnetwork/protocolmetrics.py

```diff
@@ -387,5 +387,5 @@ def results(self, request):
         if len(self.columns) > 0:
             self.columns.append("name") if "name" not in self.columns else None
             self.columns = list(set(self.columns))
-        with Timer(self._api, "Fetching {} Metrics".format(protocol)):
-            return self._filter_stats(protocol)
+        #with Timer(self._api, "Fetching {} Metrics".format(protocol)):
```

Comment:
any particular reason why timer was removed ?

Reply:
Some of the scripts were failing coz of this. Discussed with Anish and removed